APILambda/events/createEvent.test.ts

```diff
@@ -1,9 +1,18 @@
-import { conn } from "TiFBackendUtils"
+import {
+  SearchForPositionResultToPlacemark,
+  addPlacemarkToDB,
+  conn
+} from "TiFBackendUtils"
 import { callCreateEvent } from "../test/apiCallers/events.js"
 import { testEventInput } from "../test/testEvents.js"
 import { createUserFlow } from "../test/userFlows/users.js"
+import { missingAddressTestLocation } from "../test/testApp.js"
 
 describe("CreateEvent tests", () => {
+  beforeEach(() => {
+    jest.restoreAllMocks()
+  })
+
   it("should allow a user to create an event and add them to the attendee list", async () => {
     const { token, userId } = await createUserFlow()
     const createEventResponse = await callCreateEvent(token, testEventInput)
```

Comment:
Use createEventFlow instead but it needs to be modified to return a response (status code and body)